build.gradle

```diff
@@ -19,9 +19,8 @@ allprojects {
     repositories {
         google()
         jcenter()
-        maven {
-            url 'https://esri.bintray.com/arcgis'
-        }
+        maven { url 'https://esri.jfrog.io/artifactory/arcgis' }
+        maven { url 'https://olympus.esri.com/artifactory/arcgisruntime-repo/' }
```

Comment:
Which of these two urls is the public url? Is it jfrog? Am I wrong in my understanding here that first maven will try to resolve the jfrog url before falling back to the olympus build? Does this create a development pattern where gradle builds using using the public sdk before falling back on a daily build? I'm probably wrong here, just wanted to check.

Reply:
the jfrog is the public url. 
Gradle will only look for the build specified in the script. The daily builds are not available publicly on Jfrog. So when we are looking for daily builds Gradle will not find them at jfrog and will look for them at the Olympus location.

https://docs.gradle.org/current/userguide/declaring_repositories.html#sec:declaring_multiple_repositories

Reply:
> So when we are looking for daily builds Gradle will not find them at jfrog and will look for them at the Olympus location.

Given this, should we switch the order of the two dependencies such that a credentialed (internal) developer can build with the daily build, only falling back on the public distribution if off network?

Reply:
Decided with @esreli over a call that we want to keep the order as it gives preference to our external user workflow.